libwidgets/View/SearchResults.vala

```diff
@@ -70,12 +70,13 @@ namespace Files.View.Chrome {
             }
 
             public Match.from_bookmark (Bookmark bookmark, SearchResults.Category category) {
-                Object (name: Markup.escape_text (bookmark.label),
+                Object (name: Markup.escape_text (bookmark.basename),
```

Comment:
I think that this should check for both basename and custom name. 

Say I have a bookmark to `~/Documents/`, which I renamed in the sidebar to "Projects" (I know this doesn't make sense to do, it's just an example). If I search for "Projects" it won't appear in the search box. But if I search for "Documents" it'll appear in the search box as "Documents". This doesn't reflect how it appears in the sidebar, which is a little strange.

I can see some design issues here, since it would mean that the same directory could appear in multiple places in the search box under different names. But I think that for the bookmark entry, it should contain the bookmark's name.

We'd also have to consider if we want to be able to search for basename and have it appear in bookmarks in this case. So if I search for Documents, should it pull up the Projects bookmark because the path matches, even if the displayed name doesn't?

This also applies to the other places you used basename in this file.

Reply:
Latest commit implements searching and displaying both basename and, if different, the custom name of bookmarks.